src/DynamoSandbox/app.config

```diff
@@ -8,8 +8,32 @@
       <probing privatePath="dll"/>
     </assemblyBinding>
   </runtime>
+  <system.serviceModel>
```

Comment:
Config data for the Thor climate server is added here. This is the same as is found in the Revit app config.